handsontable/src/3rdparty/walkontable/src/renderer/columnHeaders.js

```diff
@@ -91,7 +91,7 @@ export default class ColumnHeadersRenderer extends BaseRenderer {
       if (this.table.isAriaEnabled()) {
         setAttribute(TR, [
           A11Y_ROW(),
-          A11Y_ROWINDEX(rowHeaderIndex + 1)
+          A11Y_ROWINDEX(-1)
```

Comment:
I don't think `rowindex` can be set to `-1`:

> The value for aria-rowindex is an integer greater than or equal to 1, greater than the aria-rowindex value of any previous rows, and less than or equal to the number of rows in the full table.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-rowindex

Reply:
@evanSe Please confirm. The specification says one thing and the [issue](https://github.com/handsontable/dev-handsontable/issues/1589) says another. I don't know what should be here.

Reply:
Reverted